app/models/journeys/get_a_teacher_relocation_payment/slug_sequence.rb

```diff
@@ -36,14 +35,7 @@ def initialize(journey_session)
       end
 
       def slugs
-        SLUGS.dup.tap do |sequence|
-          if answers.trainee?
-            sequence.delete("state-funded-secondary-school")
-            sequence.delete("contract-details")
-          else
-            sequence.delete("trainee-details")
-          end
-        end
+        SLUGS.dup
```

Comment:
do you need the `dup` here? It made sense before as it got mutated

Reply:
Not strictly here, but in future commits we do end up mutating this